ui_src/src/components/createStationForm/index.js

```diff
@@ -286,31 +287,32 @@ const CreateStationForm = ({ createStationFormRef, getStartedStateRef, finishUpd
                         </div>
                     )}
                 </div>
-                <div className="replicas-container">
-                    <TitleComponent
-                        headerTitle="Replicas"
-                        typeTitle="sub-header"
-                        headerDescription="Amount of mirrors per message."
-                        learnMore={true}
-                        link="https://docs.memphis.dev/memphis/memphis/concepts/station#replicas-mirroring"
-                    />
-                    <div>
-                        <Form.Item name="replicas" initialValue={getStartedStateRef?.formFieldsCreateStation?.replicas || actualPods[0]} style={{ height: '50px' }}>
-                            <SelectComponent
-                                colorType="black"
-                                backgroundColorType="none"
-                                borderColorType="gray"
-                                radiusType="semi-round"
-                                height="40px"
-                                popupClassName="select-options"
-                                options={actualPods}
-                                value={getStartedStateRef?.formFieldsCreateStation?.replicas || actualPods[0]}
-                                onChange={(e) => getStarted && updateFormState('replicas', e)}
-                                disabled={!allowEdit}
-                            />
-                        </Form.Item>
-                    </div>
-                </div>
+                {isCloud() ? null :
```

Comment:
Replace it with !isCloud() && <div ....

Reply:
resolve in this pr: https://github.com/memphisdev/memphis/pull/1091